docs/src/references/cli/tedge-http.md

```diff
@@ -0,0 +1,224 @@
+
+```sh title="Enforcing mTLS"
+              http.ca_path  Path to a directory containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the File Transfer Service. 
+                            Example: /etc/ssl/certs
+```
+
+### On the box running the cumulocity mapper
+
+The following `tedge config` settings control the access granted to child devices
+on the HTTP services provided by the Cumulocity mapper ([Cumulocity proxy](/references/cumulocity-proxy/)).
+This can be done along three security levels.
+
+```sh title="Listening HTTP requests"
+    c8y.proxy.bind.address  The IP address local Cumulocity HTTP proxy binds to. 
+                            Example: 127.0.0.1
+       c8y.proxy.bind.port  The port local Cumulocity HTTP proxy binds to. 
+                            Example: 8001
+```
+
+```sh title="Enabling TLS aka HTTPS"
+       c8y.proxy.cert_path  The file that will be used as the server certificate for the Cumulocity proxy. 
+                            Example: /etc/tedge/device-certs/c8y_proxy_certificate.pem
+        c8y.proxy.key_path  The file that will be used as the server private key for the Cumulocity proxy. 
+                            Example: /etc/tedge/device-certs/c8y_proxy_key.pem
+```
+
+```sh title="Enforcing mTLS"
+         c8y.proxy.ca_path  Path to a file containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the Cumulocity Proxy. 
+                            Example: /etc/ssl/certs
+```
+
+### One all boxes
+
+The following `tedge config` settings control how child devices access the local HTTP services.
+This has to be done in consistent way with the main agent and Cumulocity mapper settings.
+
+```sh title="Reaching local HTTP services"
+     c8y.proxy.client.host  The address of the host on which the local Cumulocity HTTP Proxy is running, used by the Cumulocity mapper. 
+                            Examples: 127.0.0.1, 192.168.1.2, tedge-hostname
+     c8y.proxy.client.port  The port number on the remote host on which the local Cumulocity HTTP Proxy is running, used by the Cumulocity mapper. 
+                            Example: 8001
+```
+
+```sh title="Using TLS aka HTTPS"
+              http.ca_path  Path to a directory containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the File Transfer Service. 
+                            Example: /etc/ssl/certs
+         c8y.proxy.ca_path  Path to a file containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the Cumulocity Proxy. 
+                            Example: /etc/ssl/certs
+```
+
+```sh title="Using mTLS"
+http.client.auth.cert_file  Path to the certificate which is used by the agent when connecting to external services. 
+ http.client.auth.key_file  Path to the private key which is used by the agent when connecting to external services. 
```

Comment:
There are no settings for `c8y.proxy.client.auth`. Am I missing something?

Reply:
Yeah you're right, that is a bit unexpected. Maybe the tedge-agent uses the `http.client.auth.*` properties when interacting with with the file transfer server or the c8y proxy (as the property is called `http.client.auth*`), but still it would open up for misconfiguration if the user configured the c8y.proxy (server) to use a different certificate as the tedge-agent's file transfer service (e.g. when they are not sharing certificates).